README.md

```diff
@@ -36,10 +38,17 @@ $ python my_protocol.py | transcriptic analyze
 
 ##### Submit a protocol to Transcriptic
 ```
-$ python my_protocol.py | transcriptic submit --project sequencing --title "Sequencing run"
+$ python my_protocol.py | transcriptic submit --project "sequencing" --title "Sequencing run"
 Run created: https://secure.transcriptic.com/cambridge/sequencing/r1xa043277aekj
 ```
 
+##### Run a protocol by passing it a .json file with parameters
+```
+$ transcriptic run my_protocol.py protocol_config.json
```

Comment:
pretty sure this needs to be `transcriptic run MyProtocol protocol_config.json`.